packages/web-twig/tests/__snapshots__/ComponentsSnapshotTest__test with data set textFieldDefault.twig__1.html

```diff
@@ -3,6 +3,6 @@
     <label for="example" class="TextField__label  TextField__label--required">
         Text field
     </label>
-    <input minlength="6" type="text" id="example" name="example" class="TextField__input" required>
```

Comment:
Why is this attribute missing?

Reply:
Good eyes, thanks. I forgot to pass rest of the props.